docs/api/devguide/searchableproperties.md

```diff
@@ -0,0 +1,79 @@
+# Adjust searchable feature-properties
+
+This section describes how to use the extended capability of enabling property-search for
+user-specified properties of your GeoJSON features inside a space. This API capability is available
+to users having the XYZ PRO usage-plan.
+
+It is recommended reading the ["properties-search"](propertiessearch.md) guide about how searching
+on properties works in general, before going on with this more specific guide.
+
+## Search for features in the space
+
+If the above prerequisites are fulfilled adjusting the searchable properties can be done by
+updating the space's `searchableProperties` property. This can be done using a `PATCH` request to
+the `/spaces/{spaceId}` endpoint.
+
+`searchableProperties` is a map of which the keys are property names and the values are boolean
+flags telling whether the property should be searchable or not.
+
+The following sample shows how to define `someProperty1` to be searchable and `someProperty2` to
+be not searchable. In case you're wondering about the latter: That could be necessary to tell XYZ
+to revoke the decision of making `someProperty2` searchable in the automated algorithm.
+
+*NOTE: XYZ has a space-specific algorithm to automatically decide which of its properties
+should be searchable. This applies only to top-level properties of your Features.*
+
+**TL;DR**
```

Comment:
This really feels unnecessary as you spelled it out literally in the paragraph before. I'd remove this.

Reply:
Just forgot to remove the previous paragraph. The only which actually should've been kept was the 2nd one. Fixed in followup PR.